README.md

```diff
@@ -46,15 +46,9 @@ where `sync-prefix` is `/ndn/svs` for the example application.
 
 ## Contributing
 
-We greatly appreciate contributions to the ndn-svs code base, provided that they are
-licensed under the LGPL 2.1+ or a compatible license (see below).
-If you are new to the NDN software community, please read the
-[Contributor's Guide](https://github.com/named-data/.github/blob/master/CONTRIBUTING.md)
-to get started.
```

Comment:
Deleted this paragraph because that `CONTRIBUTING.md` file explains how to submit patches via gerrit which ndn-svs does not use.